samples/client/petstore/qt5cpp/client/SWGUserApi.h

```diff
@@ -35,14 +35,14 @@ class SWGUserApi: public QObject {
     QString basePath;
     QMap<QString, QString> defaultHeaders;
 
-    void createUser(SWGUser body);
-    void createUsersWithArrayInput(QList<SWGUser*>* body);
-    void createUsersWithListInput(QList<SWGUser*>* body);
+    void createUser(SWGUser& body);
+    void createUsersWithArrayInput(QList<SWGUser*>*& body);
+    void createUsersWithListInput(QList<SWGUser*>*& body);
```

Comment:
References to pointers are a bit odd, their only purpose is to be able to modify the pointer itself in the function. Is that the intent here?

Reply:
Side note, a pointer is already a cheap copy, and doesn't benefit from passing it as a reference to pointer. You could make this a const pointer instead to state that this is a read-only input.

Reply:
Yes I agree with the pointer stuff. But the Mustache is a quite big and a bit ugly that is why  I did not add the code to do it that way. But I will check if we can do it cleanly.

Reply:
Right, I see. Nevermind my comment then, just use `const &` regardless of the type.
Agreed on the ugliness, afaict that whole qt code is in dire need of refactoring anyway, way too many raw pointer handling for my taste 😳

Reply:
Indeed. In fact I have removed all memory reallocation in the helper class. There were too many memory leaks. I will just add the const.

Reply:
Ok, it is not so small change... The object is accessed and it has to have a const specifier in the methods being invoked as well. So the changes are cascading. If you agree I will drop the const too. 
Right now I need to prevent the copy and the reference is working. Maybe I do this after I make the other PRs.

Reply:
Ok, it's definitely outside the scope of this PR then. The non-const reference will do for now :)